pt_PT.ini

```diff
@@ -10,52 +10,52 @@ Sound speed hack (DOA etc.) = Sound speed hack (Dead Or Alive, etc.)
+Analog Limiter = Limitador do analógico
+Analog Mapper High End = Mapeamento do analógico de alta qualidade (sensibilidade dos eixos)
+Analog Mapper Low End = Mapeamento do analógico de baixa qualidade (zona não coberta inversa)
+Analog Mapper Mode = Modo de mapamento do analógico
+Analog Stick = Analógico
 Auto = Auto
-Auto-centering analog stick = Auto-centering analog stick
-Button Opacity = Buttons opacity
-Button style = Button style
-Calibrate D-Pad = Calibrate
-Calibration = Calibration
-Classic = Classic
-Control Mapping = Control mapping
-Custom layout... = Edit touch control layout...
-Customize tilt = Customize tilt...
+Auto-centering analog stick = Auto-centralização do analógico
+Button Opacity = Opacidade dos botões
+Button style = Estilo dos botões
+Calibrate D-Pad = Calibrar D-Pad
+Calibration = Calibração
+Classic = Clássico
+Control Mapping = Mapeamento de Controlos
+Custom layout... = Editar layout dos controlos de toque...
+Customize tilt = Personalizar inclinação...
 D-PAD = D-Pad
-Deadzone Radius = Deadzone radius
-DInput Analog Settings = DInput analog settings
-Disable D-Pad diagonals (4-way touch) = Disable D-Pad diagonals (4-way touch)
-HapticFeedback = Haptic feedback (vibration)
-Ignore gamepads when not focused = Ignore gamepads when not focused
-Ignore Windows Key = Disable Windows key
-Invert Axes = Invert axes
-Invert Tilt along X axis = Invert tilt along X axis
-Invert Tilt along Y axis = Invert tilt along Y axis
-Keyboard = Keyboard control settings
-Landscape = Landscape
-Landscape Reversed = Landscape reversed
-None (Disabled) = None (disabled)
+Deadzone Radius = Raio não coberto
+DInput Analog Settings = Definições de analógico DInput
+Disable D-Pad diagonals (4-way touch) = Desactivar diagonais do D-Pad (4 direcções)
+HapticFeedback = Resposta háptica (vibração)
+Ignore gamepads when not focused = Ignorar controladores quando não focado
+Ignore Windows Key = Ignorar a tecla Windows
+Invert Axes = Inverter eixos
+Invert Tilt along X axis = Inverter inclinação no eixo X
+Invert Tilt along Y axis = Inverter inclinação no eixo Y
+Keyboard = Configurações de controlo do teclado
+Landscape = Paisagem
+Landscape Reversed = Paisagem invertida
+None (Disabled) = Nenhum (desactivado)
 Off = Off
-OnScreen = Controles Na tela
-Portrait = Portrait
-Portrait Reversed = Portrait reversed
-PSP Action Buttons = PSP action buttons
-Screen Rotation = Screen rotation
-Sensitivity = Sensitivity
-Show Touch Pause Menu Button = Show pause menu button
-Thin borders = Thin borders
-Tilt Input Type = Tilt input type
-Tilt Sensitivity along X axis = Tilt sensitivity along X axis
-Tilt Sensitivity along Y axis = Tilt sensitivity along Y axis
-To Calibrate = To calibrate, keep device on a flat surface and press "Calibrate".
-Visibility = Visibility
+OnScreen = Controlos no ecrã
+Portrait = Retrato
+Portrait Reversed = Retrato invertido
+PSP Action Buttons = Botões de acção da PSP
+Screen Rotation = Rotação de ecrã
```

Comment:
The context around this string is very important but I would translate it to `Rotação do ecrã` instead of the current one. But again, the context (that I am unaware) is what dictates the right form.